### ISSUE TITLE
请问RTMPMuxer.writeVideo返回-20是什么错误？

### ISSUE THREAD

**Issue by zhzc0x** [ant-media/LibRtmp-Client-for-Android#106]
先调用open方法返回的134，然后再调用writeVideo返回-20，正常应该是大于0的吧？麻烦帮忙看下是什么原因，谢谢！

**Comment by mekya** [comment]
>First call the 134 returned by the open method, and then call writeVideo to return -20. Normally, it should be greater than 0, right? Please help to find out why, thank you!

Yes, it should be greater than 0.  You can analyze the source code here to see how it should be used. 
https://github.com/ant-media/LiveVideoBroadcaster

Google translate of my response ->

是的，它应该大于0。你可以分析这里的源代码，看看它应该如何使用。
https://github.com/ant-media/LiveVideoBroadcaster
